src/serviceregistry/5_0_0/sd/system-discovery/system-discovery_sd.tex

```diff
@@ -0,0 +1,373 @@
+
+The input operation data must meet the following criteria:
+
+\begin{itemize}
+    \item With this operation a system can revoke itself.
+\end{itemize}
+
+
+\clearpage
+
+\section{Information Model}
+\label{sec:model}
+
+Here, all data objects that can be part of the \textbf{system-discovery} service are listed and must be respected by the hosting System.
+Note that each subsection, which describes one type of object, begins with the \textit{struct} keyword, which is used to denote a collection of named fields, each with its own data type.
+As a complement to the explicitly defined types in this section, there is also a list of implicit primitive types in Section \ref{sec:model:primitives}, which are used to represent things like hashes and identifiers.
+
+\msubsection{struct}{SystemRegistrationRequest}
+\label{sec:model:SystemRegistrationRequest}
+ 
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3cm} | p{3cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+authentication & \hyperref[sec:model:Identity]{Identity} & yes & The requester of the operation. \\ \hline
+metadata &\hyperref[sec:model:Metadata]{Metadata} & no & Additional information about the system. \\ \hline
+version &\pref{Version} & no & Version of the system. \\ \hline
+addresses &  \pref{List}$<$\pref{Address}$>$ & yes & Different kind of addresses of the system.  \\ \hline
+deviceName & \pref{Name} & no & Unique identifier of the device on which the system is running. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{Identity}
+\label{sec:model:Identity}
+
+An \pref{Object} which describes the identity of a system. It also contains whether the identified system has higher level administrative rights.
+
+\msubsection{struct}{Metadata}
+\label{sec:model:Metadata}
+
+An \pref{Object} which maps \pref{String} keys to primitive, \pref{Object} or list values.
+
+\clearpage
+
+\msubsection{struct}{SystemRegistrationResponse}
+\label{sec:model:SystemRegistrationResponse}
+ 
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{4.25cm} | p{4cm} | X |} \hline
+\rowcolor{gray!33} Field & Type      & Description \\ \hline
+status & \pref{OperationStatus} & Status of the operation. \\ \hline
+name & \pref{Name} & Unique identifier of the registered system. \\ \hline
+metadata & \hyperref[sec:model:Metadata]{Metadata} & Additional information about the registered system. \\ \hline
+version & \pref{Version} & Version of the system. \\ \hline
+addresses &  \pref{List}$<$\hyperref[sec:model:AddressDescriptor]{AddressDescriptor}$>$ & Different kind of addresses of the registered system.  \\ \hline
+device & \hyperref[sec:model:DeviceDescriptor]{DeviceDescriptor} & Information about the device on which the system is running. \\ \hline
+createdAt & \pref{DateTime} & System is registered at this timestamp. \\ \hline
+updatedAt & \pref{DateTime} & System is modified at this timestamp. \\ \hline
```

Comment:
was modified (The reason I prefer this, is because "is modified" can mean that the System will be modified at the given timestamp... at least for me it is misleading)